nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
This is a bad practice, generally you should avoid using with outside of package sets. Access the functions directly from lib, eg.: `lib.mkEnableOption`.

Reply:
fixed